src/js/game/LevelMVC/LevelView.js

```diff
@@ -1355,240 +1350,126 @@ module.exports = class LevelView {
 
     this.selectionIndicator = this.shadingPlane.create(24, 44, 'selectionIndicator');
 
-    frameList = [];
-
-    frameList.push("Player_001");
-    frameList.push("Player_003");
-    frameList.push("Player_001");
-    frameList.push("Player_007");
-    frameList.push("Player_009");
-    frameList.push("Player_007");
-    for (i = 0; i < 5; ++i) {
-      frameList.push("Player_001");
+    if (playerName === 'Agent') {
+      this.generateAnimations(FacingDirection.Down, -1);
+      this.generateAnimations(FacingDirection.Right, 65);
+      this.generateAnimations(FacingDirection.Up, 130);
+      this.generateAnimations(FacingDirection.Left, 195);
+    } else {
+      this.generateAnimations(FacingDirection.Down, 0);
+      this.generateAnimations(FacingDirection.Right, 60);
+      this.generateAnimations(FacingDirection.Up, 120);
+      this.generateAnimations(FacingDirection.Left, 180);
```

Comment:
Are these offsets that we expect to eventually get from the player object? i.e. instead of having a check for player name do something like player.getOffsets()?

Reply:
Yeah, I think `generateAnimations` will go on the `Player` class, then `Agent` will override its `preparePlayerSprite` to generate animations from the correct offsets.

I'm working with the artist to see if we can standardize the offsets between the different player sprite sheets but I think this will be fine for now.